src/validation/validateFencesExistence.ts

```diff
@@ -0,0 +1,72 @@
+import reportError from '../core/reportError';
+import * as glob from 'glob';
+import * as path from 'path';
+import getAllConfigs from '../utils/getAllConfigs';
+import Config from '../types/Config';
+import ConfigSet from '../types/ConfigSet';
+import dedupe from '../utils/dedupe';
+
+interface ConfigPathPair {
+    path: string;
+    config: Config;
+}
+
+export default function validateFencesExistence() {
```

Comment:
>validateFencesExistence [](start = 24, length = 23)

Can you rename this `validateRequiredFences` (for consistency with the config option).